submissions/ophen-ayman-touhami/main.py

```diff
@@ -0,0 +1,117 @@
+from multiprocessing import Pool, cpu_count
```

Comment:
Please change the folder name to match your github username.

Also is this part of python standard lib ?

Reply:
Done 👍

Reply:
A quick google search shows that it is part of standard Lib since Python 2.6
<img width="720" alt="image" src="https://github.com/geeksblabla/blanat/assets/43254081/a106cd0d-d054-4cc2-ab2d-e555f9c132cb">